lib/libutee/include/scp03_pta.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (C) 2020, Foundries Limited
+ */
+
+#ifndef __SCP03_PTA_CLIENT_H
+#define __SCP03_PTA_CLIENT_H
+
+#define PTA_SCP03_UUID { 0xbe0e5821, 0xe718, 0x4f77, \
+			{ 0xab, 0x3e, 0x8e, 0x6c, 0x73, 0xa9, 0xc7, 0x35 } }
+
+/*
+ * Enable SCP03 support on the SE
+ *
+ * in	params[0].a = operation: any positive value will also rotate the keys
```

Comment:
Detail a bit what `operation` values refer to. (

Reply:
um..." operation: any positive value will also rotate the keys" --> operation = key rotation.

maybe is not that clear. should I reword it?

Reply:
I think i was confused by the word "also". If this argument is a rotation position value, simply state it.

Reply:
Sorry, I posted my initial comment before looking at how the parameter was handle.
From the code it is pretty obvious :)  So 0 => no rotation, non 0 => rotation.

My suggestion, with a format that is more consistent with other PTA API header files:
```
 * [in]  value[0].a	0 if no key rotation, 1 if key rotation requested
```
Better be strict on the value in the API description so that API is clear, hence my suggestion to state values 0 and 1 above.

Reply:
ok